routes/api.php

```diff
@@ -14,8 +14,26 @@
 |
 */
 $version = 'v1';
+$adminBase=$version . '/admin';
+$memberBase=$version . '/member';
```

Comment:
Diese Variabeln wäre im global scope, oder?

Reply:
Vom Prinzip her schon. Wo soll ich sie also hintun? (In der ".env"-Config machen sie aber wenig Sinn...)

Reply:
Hmm, wusste ich auch nicht auf Anhieb. Bin vorher aber auf _Route Prefixing_ gestossen. Vielleicht wäre das eine elegante Variante? https://laravel.com/docs/5.0/routing#route-groups